README.md

```diff
@@ -38,6 +38,29 @@
 
 ## 2024
 
+- ECCV
+  - [Approaching Outside: Scaling Unsupervised 3D Object Detection from 2D Scene](https://arxiv.org/pdf/2407.08569v1) [`det`]
+  - [Global-Local Collaborative Inference with LLM for Lidar-Based Open-Vocabulary Detection](https://arxiv.org/pdf/2407.08931v1) [`det`]
+  - [OPEN: Object-wise Position Embedding for Multi-view 3D Object Detection](https://arxiv.org/pdf/2407.10753v1) [`det`]
+  - [SEED: A Simple and Effective 3D DETR in Point Clouds](https://arxiv.org/pdf/2407.10749v1) [`det`]
+  - [General Geometry-aware Weakly Supervised 3D Object Detection](https://arxiv.org/pdf/2407.13748v1) [`det`; [PyTorch](https://arxiv.org/pdf/2407.13748v1)]
```

Comment:
PyTorch link may be https://github.com/gwenzhang/GGA ?

Reply:
yeah, I will push a new commit to fix it.